src/__init__.py

```diff
@@ -2,30 +2,30 @@
 
 # the event loop and basic configurations are initialized in env, so import it first
 # some logger configurations are set in log, so import it second to make them effective in child processes
-from . import env, log
-from . import aio_helper
+from . import aio_helper, env, log
```

Comment:
These two imports cannot be merged. The sequence is significant. Automatic import optimization could potentially break things. Please drop this commit.